File: src/main/java/me/kuku/simbot/pojo/WeiboPojo.java
package me.kuku.simbot.pojo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class WeiboPojo {
    private Long id;
    private String name;
    private String userId;
    private String created;
    private String text;
    private String bid;
    private List<String> imageUrl;
    private Boolean isForward;
    private String forwardId;
    private String forwardTime;
    private String forwardName;
    private String forwardText;
    private String forwardBid;

    public WeiboPojo(String name, String userId){
        this.name = name;
        this.userId = userId;
    }
}

File: src/main/kotlin/me/kuku/yuq/pojo/WeiboPojo.kt
package me.kuku.yuq.pojo

data class WeiboPojo(
        var id: String = "",
        var name: String = "",
        var userId: String = "",
        var created: String = "",
        var text: String = "",
        var bid: String = "",
        var imageUrl: List<String> = listOf(),
        var isForward: Boolean = false,
        var forwardId: String = "",
        var forwardTime: String = "",
        var forwardName: String = "",
        var forwardText: String = "",
        var forwardBid: String = ""
)